inputs-required.tf

```diff
@@ -1,23 +1,14 @@
 variable "env" {
-  description = "Environment value."
+  description = "Environment value"
```

Comment:
can we put these changes into the base terraform template?

Reply:
https://github.com/hmcts/terraform-module-template/commit/d42e1b740d31064495c533be4a4daec8f44128af